src/librustc_driver/driver.rs

```diff
@@ -473,27 +453,29 @@ pub fn phase_2_configure_and_expand(sess: &Session,
     });
 
 
-    krate = time(time_passes, "crate injection", ||
-                 syntax::std_inject::maybe_inject_crates_ref(krate,
-                                                             sess.opts.alt_std_name.clone()));
+    krate = time(time_passes, "crate injection", || {
```

Comment:
Seems to be doing the same as the ones above, but not below. I think because the closure has a block it is more eager to put it across multiple lines, and then we can get the other args on one line.